### ISSUE TITLE
Region-based navigation

### ISSUE THREAD

**Issue by gregoriopellegrino** [mattgarrish/epub-revision#2914]
### Section

Region-based navigation

### Describe the problem

The Region-based navigation section refers to [EPUB Region-Based Navigation 1.0](https://idpf.org/epub/renditions/region-nav/epub-region-nav-20150826.html). Is this specification still active? Has it been adopted?

**Comment by sueneu** [comment]
Widespread support for region based navigation would be great thing for people who like comics.

As far as I know, there has been little reading system support for it. 

Ken Jones might be good to talk to. His authoring product, CircularFlo, will make region based navigation but the only RS I know of that supports it is Colibrio.

One big obstacle to widespread adoption of this navigation standard is that Amazon has a [propriety technology that does the same thing.](https://www.amazon.com/Kindle-Comic-Creator/b?ie=UTF8&node=23496309011) It would be a burden on publishers to create two version of a comic with two different navigation technologies. 

For instance:
Some of the comics we have in studio are 200+ pages with 4, 5 or more panels per page. Auto generating only goes so far with most titles. Any title where art or text breaks a panel edge requires a person to make a decision and create the navigational regions. This can be a significant cost in time and labor.

I hope I am wrong about RS support.

**Comment by gregoriopellegrino** [comment]
The fact is that this is an IDPF document, I'm not sure it's a W3C specification that we can refer to...

**Comment by sueneu** [comment]
Agreed. Apologies if I missed the focus of your comment. Where in the documentation are we referring to this?

**Comment by gregoriopellegrino** [comment]
Here: https://htmlpreview.github.io/?https://github.com/w3c/epub-specs/blob/fxl-a11y-tech-restructure/wg-notes/fxl-a11y-tech/index.html

**Comment by HadrienGardeur** [comment]
With what I'm currently pushing for in SMIL (see https://github.com/w3c/epub-specs/pull/2919 and https://github.com/w3c/epub-specs/pull/2883), I don't think that region-based navigation would be required as a separate spec anymore.

Take a look at [the example that I've created](https://github.com/HadrienGardeur/accessible-epub-comics/) and more specifically [the SMIL for the first page](https://github.com/HadrienGardeur/accessible-epub-comics/blob/main/EPUB/page1.smil).

With this SMIL, you get:

- panel by panel navigation
- textual and audio descriptions for panels
- textual and audio equivalent for `textOnVisual`

Since `<audio>` is optional with our content model in EPUB, the only requirement would be a textual equivalent.

There are some subtle differences though:

- region-based navigation also supports [circular](https://idpf.org/epub/renditions/region-nav/epub-region-nav-20150826.html#sec-3.5.1.2) and [polygonal](https://idpf.org/epub/renditions/region-nav/epub-region-nav-20150826.html#sec-3.5.1.3) fragments in addition to rectangles
- it can also convey [very minimal metadata](https://idpf.org/epub/renditions/region-nav/epub-region-nav-20150826.html#sec-3.6.2) (color and character)
- and provides supports for [regions that require a synthetic spread](https://idpf.org/epub/renditions/region-nav/epub-region-nav-20150826.html#sec-3.5.2)

**Comment by avneeshsingh** [comment]
copying discussions from Feb 19 meeting for ecord purpose.
Avneesh Singh: This section is pointing to old IDPF spec.

Gregorio Pellegrino: region based spec was defined 11years ago.
… especially to fixed layout, zooming in/out and combo with media overlays
… is this an active spec? is it still valid, can we reference it?

Avneesh Singh: its quite risky to reference to the IDPF page and it hasn't migrated to W3C space.

Charles LaPierre: there is a case maybe to moving it over to W3C?

George Kerscher: I agree if we are going to ref it it should be moved over and vetted.

Avneesh Singh: do we really need it, if so we will ask Matt/Ivan to move it.

Avneesh Singh: yes lets ask @mattgarrish to review it

**Comment by mattgarrish** [comment]
> … is this an active spec? is it still valid, can we reference it?

You can reference anything you want as you're only producing a note.

But you can't recommend technologies that don't have widespread support as meeting accessibility requirements, so just be careful what guidance you're giving people about its viability.

**Comment by reidmore-online** [comment]
Proposal: Move Region-based Navigation to a CG Report, keeping a record of it for incubation purposes and making it linkable to the FXL documents.
